aws/k8s/cluster.tf

```diff
@@ -0,0 +1,6 @@
+module "my-cluster" {
+  source = "github.com/env0/k8s-modules//aws"
```

Comment:
why double slash `/`? 
`//aws`

Reply:
This TF thing - means folder in the repo

Reply:
@razbensimon It's for using a git provider as a source, and picking a specific folder in there
See https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories